src/io/mod.rs

```diff
@@ -269,13 +269,15 @@
 //! [`Result`]: https://doc.rust-lang.org/std/result/enum.Result.html
 //! [`.unwrap()`]: https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap
 
+const DEFAULT_BUF_SIZE: usize = 8 * 1024;
```

Comment:
should this const be marked `pub(crate)`? Is that even possible?

I guess I'm a bit confused how you're accessing this from other crates without it having a visibility specifier haha. Feel like I'm missing something about how consts work :P

Reply:
Well, I think sub mod always can access father mod.

Reply:
Just like write `tests` in sub mod even for test some private functions.
I think pub(crate) is for being public to current crate and private to extern crate.

Reply:
See `std::io`,  I just imitate its way of writing.
![image](https://user-images.githubusercontent.com/16250688/68958984-2c0c8500-0808-11ea-97f1-cb8dc06909aa.png)